README.md

```diff
@@ -9,7 +9,7 @@
 
 - HTML/CSS3 (SASS)
 - Javascript
-- Linters
+- Linters (Webhint, Eslint, Stylelint)
 - Visual Studio
 
 ## Live Demo (if available)
```

Comment:
- [ ]  Your project  should provide a Loom recorded vidoes based on the following : 
- record a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, showing the pages in different screen sizes.
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
> * [ ]   Your project  should provide a Loom recorded vidoes based on the following :
> * record a video presenting the features of the project.
> 
> 2. The maximum length of the video is 5 minutes.
> 3. The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
> 4. The video contains a demo of the project, showing the pages in different screen sizes.
> 5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

- I fixed the top bar display. Actually, it was there but hidden and it is fixed now.
- I also added the video presentation link.